drivers/storage/portworx/component/telemetry.go

```diff
@@ -1180,9 +1185,16 @@ func readConfigMapDataFromFile(
 }
 
 func getCCMListeningPort(cluster *corev1.StorageCluster) int {
+	defCCMPort := defaultCCMListeningPort
+
+	pxVer30, _ := version.NewVersion("3.0")
+	if pxutil.GetPortworxVersion(cluster).GreaterThanOrEqual(pxVer30) {
+		defCCMPort = defaultCCMListeningPortForPXge30
+	}
+
 	startPort := pxutil.StartPort(cluster)
 	if startPort == pxutil.DefaultStartPort {
-		return defaultCCMListeningPort
+		return defCCMPort
```

Comment:
All our ports (including the telemetry port) should be moved respectively if a user provides a custom start port

Reply:
@piyush-nimbalkar we are moving the "startPort + 20" which aligns with the docker-entrypoint shift.

Reply:
Talked to @jrivera-px offline. We will continue with incremental port increases in case of custom start port. As we have been using `startPort + 20` for log uploaded so far, we will continue with that. Let's not create 'holes' at least in the custom port range.